variables.tf

```diff
@@ -5,4 +5,5 @@ variable "region" {
 variable "repo_name" {}
 variable "repo_connection" {}
 variable "discord_app_id" {}
-variable "discord_public_key" {}
\ No newline at end of file
+variable "discord_public_key" {}
+variable "rawg_api_key" {}
```

Comment:
I'll put the api key in GCP manually after these changes have been deployed so we don't need to pass it through a variable.
As an aside, secrets passed through variables in terraform should be marked as sensitive: https://developer.hashicorp.com/terraform/tutorials/configuration-language/sensitive-variables#refactor-database-credentials
```suggestion
```

Reply:
Do we need the same variable in infra/variables.tf?

Reply:
No, we can get the secret id directly from the secret resource